babel-watch.js

```diff
@@ -247,18 +249,21 @@ function restartApp() {
     }
   }
 
-  // Support for --debug option
   const runnerExecArgv = process.execArgv.slice();
-  if (program.debug) {
-    runnerExecArgv.push('--debug=' + program.debug);
-  }
-  // Support for --inspect option
-  if (program.inspect) {
-    runnerExecArgv.push('--inspect');
-  }
-  // Support for --debug-brk
-  if(program.debugBrk) {
-    runnerExecArgv.push('--debug-brk');
+  const unknownOptions = program.parseOptions(program.normalize(process.argv.slice(2))).unknown;
+
+  if (unknownOptions.length > 0) {
+    for (let i = 0, len = unknownOptions.length; i < len; ++i) {
```

Comment:
This block is required, and I add it intentionally. 
When just using `runnerExecArgv = runnerExecArgv.concat(unknownOptions);`, and run 
`babel-watch --harmony --inspect -w ./src -D --debug=2388 ./src/server.js`

Args will be parsed to `[ '--harmony', '--inspect', '--debug', '2388' ]`,  and it causes an error: 
`Error: Cannot find module '/Users/kouhin/git/myproj/2388'`